libcloud/compute/drivers/profitbricks.py

```diff
@@ -670,10 +696,10 @@ def create_node(
         :type   ex_disk: ``int``
 
         :param  ex_password: The password for the volume.
-        :type   ex_password: ``str``
+        :type   ex_password: :class:`NodeAuthPassword`
 
         :param  ex_ssh_keys: Optional SSH keys for the volume.
-        :type   ex_ssh_keys: ``list`` of ``str``
+        :type   ex_ssh_keys: ``list`` of :class:`NodeAuthSSHKey`
```

Comment:
Introducing NodeAuthPassword and NodeAuthSSHKey sounds like a good idea. However, changing the expected type or ex_password and ex_ssh_keys is a breaking change, right? Why don't you create the objects from the `str` parameters?

Reply:
Right, this would be a breaking change. We thought it's more common/convenient for the users to pass `NodeAuthSSHKey` and `NodeAuthPassword` instances. I'm going to revert it to `str`, except you have a better suggestion?

Reply:
You can also decide to accept both. I don't really know what's the best thing to do here, but I would personally rather avoid breaking code for some small convenience improvement. (I'm not a libcloud maintainer, though.)